go/vt/vtgate/vstream_manager.go

```diff
@@ -683,6 +691,26 @@ func (vs *vstream) streamFromTablet(ctx context.Context, sgtid *binlogdatapb.Sha
 	}
 }
 
+// isRetriable determines whether we should exit immediately or retry the vstream.
+// The first return value determines if the error is retriable, the second indicates whether
+// the tablet on which the error occurred should be ommitted from the candidate list of tablets
+// to choose from on the retry.
+func (vs *vstream) isRetriableError(err error) (bool, bool) {
```

Comment:
Nit, but this should be `Retryable`. And I think `shouldRetry()` might be a better name.